apps/factory/src/components/prompt_version.tsx

```diff
@@ -265,14 +277,23 @@ function PromptVersion({
               />
             )}
 
-            <Button
+            <LoadingButton
               color="success"
               variant="outlined"
               onClick={handleRun}
               disabled={template.length <= 10}
+              loadingPosition="end"
```

Comment:
mark it start

Reply:
I have made relevant changes and also made a few changes in update package component and prompts/index.ts file